tests/bin/check_tron_jobs_test.py

```diff
@@ -1403,6 +1403,23 @@ def test_sort_runs_by_interval_day_empty_buckets(self):
         assert '2018.10.14' in run_buckets
         assert run_buckets['2018.10.14'] == []
 
+    @patch('check_tron_jobs.Client', autospec=True)
+    def test_compute_check_result_for_job_not_precious(self, mock_client):
+        client = mock_client('fake_server')
+        del self.job['monitoring'][check_tron_jobs.PRECIOUS_JOB_ATTR]
+        check_tron_jobs.guess_realert_every = mock.Mock(return_value=1)
+        check_tron_jobs.get_object_type_from_identifier = \
+            mock.Mock(return_value=mock.Mock())
+        client.job = mock.Mock(return_value=self.job)
+        check_tron_jobs.compute_check_result_for_job_runs = mock.Mock(
```

Comment:
Can you also assert that compute_check_result_for_job_runs is only called once in the non-precious case?

Reply:
Sure.